packages/graphback-codegen-client/src/ClientCRUDPlugin.ts

```diff
@@ -41,28 +41,27 @@ export const CLIENT_CRUD_PLUGIN = "ClientCRUDPlugin";
  */
 export class ClientCRUDPlugin extends GraphbackPlugin {
     private pluginConfig: ClientGeneratorPluginConfig;
-
+     
     constructor(pluginConfig?: ClientGeneratorPluginConfig) {
         super()
         this.pluginConfig = Object.assign({ outputFormat: 'ts' }, pluginConfig);
 
     }
-    // TODO change interface to return metadata instead of schema
-    public transformSchema(metadata: GraphbackCoreMetadata): GraphQLSchema {
-        const schema = metadata.getSchema()
+
+    public init(metadata: GraphbackCoreMetadata) {
```

Comment:
So we have this as a public method, but purely for testing purposes? I guess I find this a bit unusual. A method should not be public just so that tests can use it IMO.

I would also never expect a method named `init` to return data.

Reply:
This can be used in instrumentation tests on top that I'm building where we do not want to polute filesystem. We have been doing a lots of testing with filesystem but that is kinda tricky. I'm going to use this pattern for testing, but later maybe we can figure out something else - like tests can extend plugin etc :)